app/src/androidTest/java/vn/asiantech/internship/AddNoteTest.java

```
@@ -0,0 +1,128 @@
+package vn.asiantech.internship;
+
+import android.support.test.espresso.Espresso;
+import android.support.test.espresso.action.ViewActions;
+import android.support.test.espresso.assertion.ViewAssertions;
+import android.support.test.espresso.matcher.RootMatchers;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.util.Log;
+
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import vn.asiantech.internship.ui.main.NoteActivity;
+
+/**
+ * @author at-cuongcao
+ * @version 1.0
+ * @since 7/13/2017.
+ */
+@RunWith(AndroidJUnit4.class)
+public class AddNoteTest {
+
+    private static final String TAG = "tag11";
+
+    @Rule
+    public ActivityTestRule<NoteActivity> mActivityTestRule = new ActivityTestRule<NoteActivity>(NoteActivity.class);
```

@at-trungnguyen 
Please ignore this error and same after.
Thank you :)